packages/web-app-files/src/spaces/views/Spaces.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="oc-p-s">
+    <h2 v-text="$gettext('Spaces')" />
+    <div>
+      <span v-text="$gettext('Access all project related files in one place.')" />
+      <a href="#" v-text="$gettext('Learn more about spaces.')" />
+      <h3 v-text="$gettext('Your spaces')" />
+      <hr class="oc-mb-s" />
+    </div>
```

Comment:
The div should wrap the entire content of the spaces list. I.e. span, a, h3, hr must be on the same level as the list-loader and the template.